### ISSUE TITLE
Candle close bid strategy instead of ticker price bid (or ask) side strategy

### ISSUE THREAD

**Issue by rylos** [freqtrade/freqtrade#3986]
## Describe your environment
(if applicable)

  * Operating system: _Ubuntu___
  * Python Version: __3.8___ (`python -V`)
  * CCXT version: _____ (`pip freeze | grep ccxt`)
  * Freqtrade Version: _dev___ (`freqtrade -V` or `docker-compose run --rm freqtrade -V` for Freqtrade running in docker)

## Describe the enhancement

The current configurable ask and bid strategy take ticker data and current bid or ask side of the price to place the buy/sell order.
I would like to place the order on the candle close (so new candle open) value. I think backtesting and hyperopts use this price and not the current ask/bid side of price (that can be slighty different).

**Comment by xmatthias** [comment]
Backtesting / hyperopt uses candle's open, not close.

Apart from this - what you're asking  will not work in real trading (it will work in dry-run as that assumes all orders fill immediately, independent of the price provided - but will fail as soon as you need someone to match your order).

Your price should be close to the current trade action. The candle's previous close may or may not be close to this, so this would lead to a lot of canceled orders - so you had a good signal, but you're missing out on the trade as you order get's never filled.

To summarize, it's currently not possible - and we will also not include this an explicit option, as it will lead to a lot of user pain due to many cancelled orders.

It may become possible "accidentally" once strategy can control the order price (#848, #909)  - but it'll require you to write the logic to use the correct price.
However, that's a long shot and not something we're currently working on.

**Comment by rylos** [comment]
It is exactly what I'm looking for. Any ETA or work in progress?

**Comment by xmatthias** [comment]
we do not give ETA's. 
This is an open source project, where people donate their free time to improve the bot.

If you think this a must for you, feel free to contribute this. 
Obviously no one before you thought it important enough to spend their free time on it, and honestly, i think there are more important (and more interresting) improvements to be made to the bot, so i will most definitely not work on this for the forseeable future.
